.github/workflows/release.yaml

```diff
@@ -1,62 +1,48 @@
 name: Release
 on:
-  # disable deploys until this action is amended to include seeding cloudflare IPFS
-  # schedule:
-  #   - cron:  '0 12 * * 1-4' # every day 12:00 UTC Monday-Thursday
-
+  schedule:
+    - cron:  '0 12 * * 1-4' # every day 12:00 UTC Monday-Thursday
   # manual trigger
   workflow_dispatch:
 
 jobs:
-  bump_version:
-    name: Bump Version
+  wait-on-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jitterbit/await-check-suites@v1
+        onlyFirstCheckSuite: true
```

Comment:
can you tell me what this flag does?

Reply:
this only waits for the first check suite. this is meant to wait for the test suite generated on merge or push, as opposed to including any suites that are run afterwards, which may include other things (crowdin, or other failed releases).